bugwarrior/services/github.py

```diff
@@ -324,16 +325,23 @@ def get_owner(self, issue):
         if issue[1]['assignee']:
             return issue[1]['assignee']['login']
 
+    def filter_issues(self, issue):
+        _, (_, issue) = issue
+        return self.filter_repo_name(os.path.basename(issue['repository_url']))
```

Comment:
Unfortunately, search results are really sparse and this seems to be the best way to get the name of the repo :( .